src/public/serviceHostKeys.ts

```diff
@@ -53,9 +32,9 @@ class ServiceElement {
       return (
         acc +
         `
-          <div class="enteredKey">
-            ${key}
-            <input type="text" value=""></input>
+          <div class="enteredKeys">
+            <span id="serviceKeyName">${key}</span>
```

Comment:
don't use Ids. You can have many providers. each page should only have 1 element with id attribute, so don't use ids